registrations/tests/test_utils.py

```diff
@@ -67,10 +73,72 @@ def test_create_ics_file_content_with_start_time_as_end_time():
         start_time=localtime(),
         end_time=None,
     )
-    (filename, ics) = create_event_ics_file_content(event)
+    filename, ics = create_event_ics_file_content(event)
 
     assert filename == "event_helsinki:123.ics"
     assert (
         ics
-        == b"BEGIN:VCALENDAR\r\nVERSION:2.0\r\nPRODID:-//linkedevents.hel.fi//NONSGML API//EN\r\nBEGIN:VEVENT\r\nSUMMARY:Event name\r\nDTSTART;TZID=Europe/Helsinki:20240101T020000\r\nDTEND;TZID=Europe/Helsinki:20240101T020000\r\nDESCRIPTION:Event description\r\nLOCATION:Place name\\, Streen address\\, Helsinki\r\nEND:VEVENT\r\nEND:VCALENDAR\r\n"
+        == b"BEGIN:VCALENDAR\r\nVERSION:2.0\r\nPRODID:-//linkedevents.hel.fi//NONSGML "
+        b"API//EN\r\nBEGIN:VEVENT\r\nSUMMARY:Event name\r\nDTSTART;TZID=Europe/"
+        b"Helsinki:20240101T020000\r\nDTEND;TZID=Europe/Helsinki"
+        b":20240101T020000\r\nDESCRIPTION:Event description\r\nLOCATION:Place name\\, "
+        b"Streen address\\, Helsinki\r\nEND:VEVENT\r\nEND:VCALENDAR\r\n"
     )
+
+
+@pytest.mark.parametrize(
+    "language,name_value,expected_fallback_language",
+    [
+        ("fi", "Event name", "en"),
+        ("fi", "Evenemangets namn", "sv"),
+        ("fi", None, "en"),
+        ("sv", "Event name", "en"),
+        ("sv", "Tapahtuman nimi", "fi"),
+        ("sv", None, "en"),
+        ("en", "Tapahtuman nimi", "fi"),
+        ("en", "Evenemangets namn", "sv"),
+        ("en", None, "fi"),
+    ],
+)
+@pytest.mark.freeze_time("2024-01-01")
+@pytest.mark.django_db
+def test_create_ics_file_using_fallback_language(
```

Comment:
There's a lot of logic 🖖 in this test. I'd suggest to split those in separate tests even if it would seem silly.

Reply:
Split this into two tests.